sdk/python/pvsite_datamodel/sqlmodels.py

```diff
@@ -113,10 +113,17 @@ class ForecastSQL(Base, CreatedMixin):
         index=True,
     )
 
+    # The timestamp at which we are making the forecast. This is often referred as "now" in the
+    # modelling code.
+    # Note that this could be very different from the `created_utc` time, for instance if we
+    # run the model for a given "now" timestamp in the past.
+    timestamp_utc = sa.Column(sa.DateTime, nullable=False, index=True)
+
     forecast_version = sa.Column(sa.String(32), nullable=False)
 
     # one (forecasts) to many (forecast_values)
     forecast_values: List["ForecastValueSQL"] = relationship("ForecastValueSQL")
+    site = relationship("SiteSQL", back_populates="forecasts")
```

Comment:
I thouight we decided to take this out, and move it for `forecast_value` table

Reply:
See other comment: the `site_uuid` is already in here, we just didn't have the corresponding `relationship`.